plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -103,19 +123,25 @@ object ScalafmtPlugin extends AutoPlugin {
   }
 
   private def formatSources(
-      cacheDirectory: File,
+      cacheStoreFactory: CacheStoreFactory,
       sources: Seq[File],
       config: Path,
       log: Logger,
       writer: PrintWriter
   ): Unit = {
-    cached(cacheDirectory, FilesInfo.lastModified, config) { cacheMisses =>
-      val changed = cacheMisses.filter(_.exists)
-      if (changed.size > 0) {
-        log.info(s"Formatting ${changed.size} Scala sources...")
-        formatSources(changed.toSeq, config, log, writer)
-      }
-    }(sources.toSet).getOrElse(())
+    trackSourcesAndConfig(cacheStoreFactory, sources, config) {
+      (outDiff, configChanged, prev) =>
+        log.debug(outDiff.toString)
```

Comment:
Missed debug logging?

Reply:
Not sure what you mean. This is a debug logging that will be displayed at debug level.

Reply:
Oh, sorry, I didn't pay attention to the logging level